src/super_gradients/training/models/prediction_pose_estimation_results.py

```diff
@@ -0,0 +1,321 @@
+import os
+from dataclasses import dataclass
+from typing import List
+
+import numpy as np
+
+from super_gradients.training.models.prediction_results import ImagePrediction, ImagesPredictions, VideoPredictions
+from super_gradients.training.models.predictions import PoseEstimationPrediction
+from super_gradients.training.utils.media.image import show_image, save_image
+from super_gradients.training.utils.media.video import show_video_from_frames, save_video
+from super_gradients.training.utils.visualization.poses import draw_skeleton
+
+
+@dataclass
+class ImagePoseEstimationPrediction(ImagePrediction):
```

Comment:
why is this file under the `models` directory? 
It looks like it belongs in some utils dir

Reply:
Maybe. TBH, I haven't though much about the file location and placed it where the rest of the files related to predict() for object detection were placed:
![image](https://github.com/Deci-AI/super-gradients/assets/532320/baabaf85-9f3b-4b52-837d-41cbee17be0e)

Reply:
Moved to super_gradients.training.utils.predict in 16c29cc93eb8827a32626b4a9f8433a0f5b818ba